tgui/packages/tgui/global.d.ts

```diff
@@ -0,0 +1,95 @@
+interface ByondType {
```

Comment:
We're using TypeScript? 🤔

Reply:
This is one of those moments, where sprinkling some TS was really necessary for VSCode to pick up some knowledge of the global objects. One day, we will embrace TS.